english/marshmallow.md

```diff
@@ -115,7 +115,7 @@ Something lightly brushes the back of your neck, sending a chill down your spine
 
 Startled, you immediately whirl around to see who (or what) touched you...but there's no one behind you.
 
-Click [here](./sonic-screwdriver/sonic-screwdriver.md) to use your sonic screwdriver to scan the air 
```

Comment:
Sorry again.  This should take care of it.

https://github.com/bynumjj/create-your-own-adventure/tree/prostate <https://github.com/bynumjj/create-your-own-adventure/tree/prostate>


> On Nov 11, 2015, at 3:55 PM, Caroline Buckey <notifications@github.com> wrote:
> 
> In english/marshmallow.md <https://github.com/udacity/create-your-own-adventure/pull/2685#discussion_r44605010>:
> 
> > @@ -115,7 +115,7 @@ Something lightly brushes the back of your neck, sending a chill down your spine
> >  
> >  Startled, you immediately whirl around to see who (or what) touched you...but there's no one behind you.
> >  
> > -Click [here](./sonic-screwdriver/sonic-screwdriver.md) to use your sonic screwdriver to scan the air 
> You have made this pull request from the prostate branch, not the master branch, of your repository. It doesn't hurt to push the commit to your master branch, but you'll need to also push it to your prostate branch in order to update this pull request.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/udacity/create-your-own-adventure/pull/2685/files#r44605010>.
>